tests/CommonTests.cpp

```diff
@@ -159,7 +159,7 @@ enum class EClassTest
 };
 DEFINE_ENUM_FLAG_OPERATORS(EClassTest);
 
-enum ERawTest
+enum ERawTest : unsigned int

```

Comment:
Clearing flags is UB if there's no explicit underlying type because `~val` will set bits beyond the representation of the enum values. There's no issue with `enum class` as it has well-defined defaults for the underlying type.